docs/release-notes/saros-i_0.1.0.md

```diff
@@ -0,0 +1,234 @@
+
+
+### Report a Bug
+
+If you encounter any other bugs not mentioned above, we would appreciate it if you would report them to our issue tracker (after checking that they have not already been reported).
+
+Our current bug tracker can be found on our [GitHub page](https://github.com/saros-project/saros/issues).
+Please make it clear that the issue is dealing with Saros/I.
+
+When reporting a bug that concerns the plugin behavior, please provide the Saros log file (or all relevant excerpts) for a session where the bug was encountered.
+The log files can be found in the IntelliJ settings directory, which usually resides in the home directory under `~/.IdeaXXXXXXXX/` (".IdeaXXXXXXXX" designates the used IntelliJ release; for IntelliJ IDEA Community Edition 2018.2, this would be `.IdeaIC2018.2`).
+The log files are located in `~/.IdeaXXXXXXXX/system/log/SarosLogs/`.
+
+If you are encountering IntelliJ IDEA errors connected to Saros (which will be displayed by a red, blinking symbol in the bottom right corner of the IntelliJ project view; the error can be viewed in more detail by clicking the symbol), please also include the IntelliJ IDEA logs.
+They are located in `~/.IdeaXXXXXXXX/system/log/` and are named `idea.log`, `idea.log.1`, etc.
+Please have a look at the contained timestamps to provide the correct file.
+
+Before attaching any log files, please make sure to redact any private information that you do not wish to make publicly available.
+
+
+## How to Use Saros/I
+
+### Adding an XMPP Account
+
+- Choose the Saros window in bottom bar.
+  - If it is not shown, click the "Saros" button in bottom bar or open it by clicking on the square on bottom left and choosing "Saros".
+- In the top bar of the Saros view, choose the "Connect" icon (connected plug icon; the left icon).
+- Choose "Add account..." from the pop-up menu.
+- Enter the fully qualified user name (`USER_NAME@YOUR.DOMIAN`).
+  - If the basic Saros XMPP server is used, the domain would be `@saros-con.imp.fu-berlin.de`.
+- Click OK.
+- Enter your password.
+- Click OK.
+- Enter the XMPP server address (optional).
+  - This option should not be necessary in most cases.
+  - If no server is supposed to be specified, just leave the field empty.
+  - If a server is specified, a port has to be specified as well.
+
+**NOTE:** As mentioned in the section [missing features](#missing-features), Saros/I does currently not support the creation, management or deletion of XMPP accounts.
+As only people on your friends list can be invited to join your Saros session, you will have to create an account and add friends to your friends list through a different client.
+The eclipse version of Saros (Saros/E) is recommended (but not mandatory) for this purpose.
+
+If you accidentally made a typo while entering your username or password, the created account entry can also only be changed or deleted through other tools.
```

Comment:
You can just delete the account store, I would not count the Windows FileExplorer or the shell as a tool.

Reply:
It's been a while since I wrote this section, but I think I meant that you can use Saros/E to fix the account store. But this is restricted to Saros/E as it is our custom account store, so "other tools" is just confusing.

I adjusted the comment to state to use Saros/E or delete the account store `~.saros/config.dat`.